### DISCUSSION TITLE
Source code of example demo

### DISCUSSION THREAD

**Discussion by ccfiel** [yesvelte/yesvelte#417] [Q&A]
Where can I find the git repo or source code of the example demo? Thanks in advance!

**Comment by pournasserian** [comment]
You can find the source of documentation and demo pages here:

- demo: src/routes/demo
- documentation: src/routes/demo

Here is the demo folder:
https://github.com/yesvelte/yesvelte/tree/dev/src/routes/demo

**Reply by calionauta** [reply]
@pournasserian, it would be great if the link to each folder were on the page of each example demo. What do you think?

**Reply by TheHadiAhmadi** [reply]
@renatocaliari yes it is good idea
